### PR TITLE
chore: remove dead code and pygithub from runtime

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -35,22 +33,19 @@ jedi==0.16.0
...
-PyGithub==1.47
 Pygments==2.6.1
-PyJWT==1.7.1
 pyparsing==2.4.6
 python-dotenv==0.12.0
 python-json-logger==0.1.11
 pytz==2019.3
 PyYAML==5.3.1
-redis==3.3.11
+redis==3.4.1
```
Comment (jd) [inline requirements.txt]: looks unrelated (and maybe wrong)

Comment (sileht) [reply requirements.txt]: Thx for caching this !!!!
